components/brave_wallet_ui/common/slices/endpoints/token_balances.endpoints.ts

```diff
@@ -446,6 +450,46 @@ export const tokenBalancesEndpoints = ({
           id: [arg.coin, arg.chainId, arg.address].join('-')
         }
       ]
+    }),
+    getBitcoinBalances: query<
+      BitcoinBalances,
+      { accountID: BraveWallet.AccountId }
```

Comment:
this can simplified to type `BraveWallet.AccountId` since there is only one argument

Reply:
Updated!